descartes_utilities/src/ros_conversions.cpp

```diff
@@ -0,0 +1,107 @@
+/*
+ * Software License Agreement (Apache License)
+ *
+ * Copyright (c) 2016, Southwest Research Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/*
+ *  ros_conversions.cpp
+ *
+ *  Created on: Feb 28, 2016
+ *  Author: Jonathan Meyer
+ */
+
+#include "descartes_utilities/ros_conversions.h"
+#include <algorithm>
+#include "console_bridge/console.h"
+
+static double minTime(const std::vector<double>& a, const std::vector<double>& b, double max_vel)
```

Comment:
+1
I'll add some documentation, but I meant this function as a sort of private, implementation function.